eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/item/HatCommand.java

```diff
@@ -40,12 +40,14 @@ void execute(Player player) {
             return;
         }
 
+        ItemStack singleItem = handItem.clone();
+        singleItem.setAmount(1);
+        handItem.setAmount(handItem.getAmount() - 1);
```

Comment:
to nie zrobi bimbimbambam? jeżeli item bedzie w ilości 1?

Reply:
powinno się wyzerować i być puste w eq

Reply:
luckson sprawdził